### PR TITLE
Add social share image and og tags

### PR DESCRIPTION
**Pull request by jamesnw**
## Description
Adds Open graph tags and a static share image.

## Steps to test/reproduce

Share the deploy preview at https://deploy-preview-98--oddcontrast.netlify.app/, and verify the content is as intended.

https://developers.facebook.com/tools/debug may be useful.

## Show me

Facebook
image

Elk (Mastodon client)
image

### CODE REVIEW

**Comment by SondraE** [comment]
[I want a social media image when I paste a link to oddcontrast - maybe showing the current colors/contrast?](https://trello.com/c/rw1C0Jtq/51-i-want-a-social-media-image-when-i-paste-a-link-to-oddcontrast-maybe-showing-the-current-colors-contrast)

**Review by SondraE — CHANGES_REQUESTED** [review]
This works well on Mastodon and Facebook, but not so well on Twitter and LinkedIn. We don't typically post on Facebook. Mastodon, Twitter, and LinkedIn are the ones we most need to support right now. (See screen shots)

**LinkedIn**
Screen Shot 2023-08-10 at 12 27 29 PM

**Twitter**
Screen Shot 2023-08-10 at 12 29 22 PM

Also, I'd like to switch from a double dash (--) to an en dash (–) in the description text?

**Comment by jamesnw** [comment]
@SondraE  Can you check again? I'm guessing you triggered the initial crawl, and so it's working now. This is what I'm seeing on Twitter-
image

And Linkedin-
image

I switched to an en dash in [e9e083c](https://github.com/oddbird/oddcontrast/pull/98/commits/e9e083cd7e62f671305aab9ae65dc5f5fe27301e)

**Comment by SondraE** [comment]
When I switched to my personal Twitter account it worked. 
It still didn't work in LinkedIn, but maybe there's a cashing issue?

**Review by SondraE — APPROVED** [review]

File: src/app.html [hunk 1]
```diff
@@ -5,6 +5,35 @@
     <meta name="description" content="" />
     <link rel="icon" href="%sveltekit.assets%/favicon.svg" />
     <meta name="viewport" content="width=device-width" />
+
+    <meta property="fb:app_id" content="1820980378150914" />
+    <meta property="og:locale" content="en_US" />
+    <meta property="og:title" content="OddContrast" />
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://oddcontrast.com" />
```
Comment (jgerigmeyer) [inline src/app.html:13]:
```suggestion
    <meta property="og:url" content="https://www.oddcontrast.com/" />
```

File: src/app.html [hunk 2]
```diff
@@ -5,6 +5,35 @@
...
+    <meta
+      property="og:description"
+      content="Color contrast checker with Oklch, Oklab, Display P3, and more – helping designers meet WCAG 2 accessibility standards."
+    />
+    <meta
+      property="og:image"
+      content="https://deploy-preview-98--oddcontrast.netlify.app/social.png"
```
Comment (jgerigmeyer) [inline src/app.html:20]:
```suggestion
      content="%sveltekit.assets%/social.png"
```

File: src/app.html [hunk 3]
```diff
@@ -5,6 +5,35 @@
...
+
+    <meta name="twitter:card" content="summary" />
+    <meta name="twitter:title" content="OddContrast" />
+    <meta
+      name="twitter:description"
+      content="Color contrast checker with Oklch, Oklab, Display P3, and more – helping designers meet WCAG 2 accessibility standards."
+    />
+    <meta
+      name="twitter:image"
+      content="https://deploy-preview-98--oddcontrast.netlify.app/social.png"
```
Comment (jgerigmeyer) [inline src/app.html:31]:
```suggestion
      content="%sveltekit.assets%/social.png"
```

File: src/app.html [hunk 4]
```diff
@@ -5,6 +5,35 @@
...
+    />
+    <meta name="twitter:domain" content="https://oddcontrast.com" />
```
Comment (jgerigmeyer) [inline src/app.html:33]:
```suggestion
    <meta name="twitter:domain" content="https://www.oddcontrast.com/" />
```

**Review by jgerigmeyer — APPROVED** [review]
Same as @SondraE -- not working for me on LinkedIn, but I'm guessing it's a caching issue.

**Pull request merged**